### PR TITLE
code challenge11

### PR DESCRIPTION
**Pull request by tahminaringer**

### CODE REVIEW

**Comment by tahminaringer** [comment]
- [yes] Top-level README “Table of Contents” is updated
 - [ kind of] Feature tasks for this challenge are completed
 - [no ] Unit tests written and passing
     - [ ] “Happy Path” - Expected outcome
     - [ ] Expected failure
     - [ ] Edge Case (if applicable/obvious)
 - [yes ] README for this challenge is complete
     - [yes ] Summary, Description, Approach & Efficiency, Solution
     - [yes ] Link to code
     - [yes ] Picture of whiteboard

File: javascript/code-challenges/queueWithStacks/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+Stack should contain the methods of push 
+pop and peek
+
+## Approach & Efficiency
+
+We used a whiteboard tool call AWW app that we mapped out the challenge on using the whiteboard demo.
+
+## Solution
+
+![whiteboard_class02](../assets/queue-with-stacks.png)
```
Comment (adrianhuebner) [inline javascript/code-challenges/queueWithStacks/README.md:25]: Looks really great Tahmina! The only thing that I would like to see added are tests!
